lib/css-transform.js

```diff
@@ -31,7 +31,7 @@ module.exports = function ({ config, entry, minify, outfile, watch }) {
       ]
     }),
     base64ify(process.cwd()),
-    postcssPresetEnv({browsers})
+    postcssPresetEnv({overrideBrowserslist})
```

Comment:
The warnings are gone, but I'm pretty sure this isn't a valid configuration option. Seems like `browsers` is still correct: https://www.npmjs.com/package/postcss-preset-env#browsers.

Reply:
Hmm, yea, I'm going to change it back to `browsers`. Sorry for the confusion between postcss libraries! It's not quite clear which change fixed the warnings (or if they went away on their own?).